tools/serverpod_cli/test/integration/analyzer/dart/endpoint_validation/endpoint_method_test.dart

```diff
@@ -569,6 +570,20 @@ class ExampleEndpoint extends Endpoint {
       );
     });
 
+    test('then toString reports that the type is not supported.', () {
+      var expectedToString = '''Found 1 issue.
+
```

Comment:
This is in a sense, a snapshot test. It compares the output with a pre-calculated message. It doesn't need to be this verbose because changing the message format will break this test. However this test is comprehensive because it covers a lot of the potential corner cases of the output with not too much test code.